src/test/java/io/xpring/payid/PayIDIntegrationTest.java

```diff
@@ -33,7 +31,7 @@ public void testXRPAddressForPayIDKnownAddressMainnet() throws PayIDException {
   public void testXRPAddressForPayIDKnownAddressTestnet() throws PayIDException {
     // GIVEN a Pay ID that will resolve on Testnet.
     PayIDClient payIDClient = new PayIDClient(XRPLNetwork.TEST);
-    String payID = "$dev.payid.xpring.money/hbergren";
+    String payID = "alice$dev.payid.xpring.money";
```

Comment:
We ran into some friction with this given that this is a dependency not only upon the reference implementation, but also with deploying the reference implementation to the specific VM behind this URL. Not asking you to change this, but curious to hear what you think the ideal solution would be here. Like spin up a local database that resolves this address? Not sure, feel like you'd have better perspective on this

Reply:
Ideally we'd spin up some sort of docker container that has hermetic data. 

I was generally waiting for things to stabilize before undertaking that endeavor. I think the ILP team has some framework they use for this. For XRP, we've been just using testnet accounts which have been relatively stable. 

This is what I feel is the most pragmatic approach, given that we're time constrained.

Reply:
Cool, ya I don't think it's mission critical that we address it now. Just wanted to learn best practices on this. Thanks!